lib/galaxy/schema/library_contents.py

```diff
@@ -308,3 +308,22 @@ class LibraryContentsDeleteResponse(Model):
 
 class LibraryContentsPurgedResponse(LibraryContentsDeleteResponse):
     purged: bool
+
+
+LIBRARY_ID = Union[DecodedDatabaseIdField, LibraryFolderDatabaseIdField]
+
+SHOW_RESPONSE = Union[
+    LibraryContentsShowFolderResponse,
+    LibraryContentsShowDatasetResponse,
+]
+
+CREATE_PAYLOAD = Union[
+    LibraryContentsFolderCreatePayload, LibraryContentsFileCreatePayload, LibraryContentsCollectionCreatePayload
+]
+
+CREATE_RESPOSNSE = Union[
```

Comment:
Those type aliases are nice, but for consistency with other similar type aliases in the code base you can name them for example as:

CREATE_RESPOSNSE -> AnyLibraryContentsCreateResponse